oshi-core/src/main/java/oshi/hardware/PowerSource.java

```diff
@@ -48,4 +49,31 @@
      *         or -2.0 (unlimited)
      */
     double getTimeRemaining();
+    
+    /**
+     * Power source health as a fraction of the battery's designed capacity.
+     * 
+     * @return A value between 0.0 (no capacity) and 1.0 (at designed capacity)
+     */
+    double getHealth();
+    
+    /**
+     * Maximum charge of the power source, in milliwatt hours.
+     * 
+     * @return If positive, milliwatt hours of charge. If negative, the maximum charge is unknown.
+     */
+    long getMaximumCharge();

```

Comment:
At the time I'm reviewing this I've only looked at the interface and not the implementation. I see we have a health/state of health measure which ratios current max vs. design max, and it's not clear to me which of those two this method fetches.   I recall in the issue we discussed including both design max capacity and current max capacity.   I gather with the health stat we can compute one from the other, but if we go that route we need to be clear which one this is.

If we're only including two stats I'd prefer to have design max and current max and let the user calculate the health themselves.

I also think `getMaximumCapacity()` would be better than `getMaximumCharge()` -- however, I also see we already have `getRemainingCapacity()` as a fraction of the max, and it's necessary to use different terms.  I was probably wrong to use capacity to represent the fraction (and not the mWh statistic) and I'd want the correct terms implemented on a major version change.  I'm wondering if there's a better way of presenting this info here.